.github/scripts/review_pr.py

```
@@ -28,13 +28,35 @@
 if response.status_code == 200:
     result = response.json()
+    review_feedback = result["candidates"][0]["content"]["parts"][0]["text"]
     print("### PR Review Feedback ###")
-    print(result["candidates"][0]["content"]["parts"][0]["text"])
+    print(review_feedback)
 else:
     print(f"❌ API Error {response.status_code}: {response.text}")
+    review_feedback = "❌ Error fetching review from Gemini."
+
+# Post Review Comment on PR
+GITHUB_API_URL = f"https://api.github.com/repos/{REPO}/issues/{PR_NUMBER}/comments"
```

Ruff (0.8.2)

Undefined name `REPO`

(F821)